README.md

```diff
@@ -85,51 +91,54 @@ where <options> are options that can be toggled on or off. See [Build Options](h
 
 Note: To make the use of the network, you have to disable the implicit scaling on PVC which can be done by uncommenting the portion of the code indicated in the sample when creating the queue.
 
-
 ## Required Hardware and Software
 
-
 ## PyTorch extension
-We provide a pybind wrapper of our tiny-dpcpp-nn implementation for seamless integration into PyTorch. Please refer to [tiny-dpcpp-nn pybind documentatino](https://intel.github.io/tiny-dpcpp-nn/manual/pytorch.html)
+
+We provide a pybind wrapper of our tiny-dpcpp-nn implementation for seamless integration into PyTorch. Please refer to [tiny-dpcpp-nn pybind documentation](https://intel.github.io/tiny-dpcpp-nn/manual/pytorch.html)
 
 Install the latest [ipex](https://intel.github.io/intel-extension-for-pytorch/index.html#installation?platform=gpu&version=v2.1.10%2Bxpu) via
 
-```
+```bash
 python -m pip install torch==2.1.0a0 torchvision==0.16.0a0 torchaudio==2.1.0a0 intel-extension-for-pytorch==2.1.10+xpu --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
 ```
 
 ## Tests
+
 When setting the additional flag `BUILD_REF_TEST=ON`, additional data from [tiny-dpcpp-data](https://github.com/intel-sandbox/tiny-dpcpp-data) will be downloaded.
 
 When setting the additional flag `BUILD_TORCH_TEST=ON`, the libtorch tests (`tnn_api.h`) will be built.
 
 To have all tests, run:
-```
-cmake -DTARGET_DEVICE="PVC" -DBUILD_REF_TEST="ON" -DBUILD_REF_TEST="ON" ..
+
+```bash
+cmake -DTARGET_DEVICE="PVC" -DBUILD_TEST="ON" -DBUILD_REF_TEST="ON" ..
```

Comment:
`-DBUILD_REF_TEST="ON" -DBUILD_REF_TEST="ON"` ->
`-DBUILD_TEST="ON" -DBUILD_REF_TEST="ON"`

Reply:
One more note: https://intel.github.io/tiny-dpcpp-nn/manual/build.html#build-options does not list `BUILD_TORCH_TEST` option.